lib/transmission-rss/client.rb

```diff
@@ -62,6 +65,9 @@ def add_torrent(file, type = :url, options = {})
       log_message = 'torrent-add result: ' + response.result
       log_message << ' (id ' + id.to_s + ')' if id
       @log.debug(log_message)
+      if log_message.include? "(429)"
```

Comment:
try raising this error in the `rpc` method

Reply:
please add spec (you can probably copy the `add_torrent.yml` file ("VCR cassette") and change the response code to 429 (in line 60) to simulate that response.

Reply:
I moved it to rpc.

Transmission itself isn't returning 429. Rather it is the tracker itself returning that error to Transmission when it attempts to add it. So the response code is actually 200. The 429 is passed back as a message from Transmission.

I added a cassette and spec code for that.